scripts/delete-bridges.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# Script used to delete Bridges and Processors used in E2E tests
+# Script will delete Bridges without Processors, for Bridges with Processors it will delete Processors if they are in proper state
+# Test Bridges are identified using "test-" prefix in their names
+
+# If Bridge contains Processors then only Processors are deleted by the script execution. If user wants to delete also Bridges then the script has to be executed again after some time (so the Processors can be removed first)!
+
+BRIDGES=($(curl -H "Authorization: $OB_TOKEN" -X GET $MANAGER_URL/api/smartevents_mgmt/v1/bridges | jq '.items[] | select(.name | startswith('\"testui-1f65f\"')) | .id' | tr -d \"))
```

Comment:
`testui-1f65f` seems hardcoded, shouldn't be there a variable?

Reply:
@sutaakar do you think that is better to create new script with the for cycle and in CI leave only call of one script?

Reply:
yes, it may be better
also it may be worth splitting the script into dedicated bash methods to increase readability

Reply:
I thought about the splitting the script to methods. Currently I am not familiar enough with bash methods (variables visibilities, good practice, etc.) to split it.